src/ui/map/map.md

```diff
@@ -0,0 +1,63 @@
+# Map
+
+The map has been refactored to make it easier to use. There is no html in the MapComponent.svelte components which makes
+this relaltively easy
```

Comment:
~~l~~ 😅

Wouldn't bother pointing this out but wanted a space to say this doc is great!